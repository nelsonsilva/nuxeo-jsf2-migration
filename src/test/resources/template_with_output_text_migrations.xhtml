<div xmlns="http://www.w3.org/1999/xhtml"
  xmlns:h="http://java.sun.com/jsf/html"
  xmlns:nxu="http://nuxeo.org/nxweb/util">

  <h3>
    <h:outputText value="#{messages['title.createGroup']}" />
  </h3>

  <div>
    <h:outputText rendered="#{not empty field}">
      <nxu:set var="canSee"
        value="#{docSuggestionActions.getDocumentExistsAndIsVisibleWithId(field)}">
        <h:outputText rendered="#{canSee}">
          <nxu:set var="doc"
            value="#{docSuggestionActions.getDocumentWithId(field)}">
            <h:outputText value="#{nxd:titleOrId(doc)}" />
          </nxu:set>
        </h:outputText>
        <h:outputText rendered="#{not canSee}"
          value="#{messages['label.documentSuggestion.docNotFoundOrNotVisible']} (#{field})" />
      </nxu:set>
    </h:outputText>
  </div>
</div>
