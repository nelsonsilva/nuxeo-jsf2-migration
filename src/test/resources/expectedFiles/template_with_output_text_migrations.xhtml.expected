<?xml version="1.0" encoding="UTF-8"?>

<div xmlns="http://www.w3.org/1999/xhtml" xmlns:h="http://java.sun.com/jsf/html" xmlns:nxu="http://nuxeo.org/nxweb/util">  
  <h3> 
    <h:outputText value="#{messages['title.createGroup']}"/> 
  </h3>  
  <div> 
    <h:panelGroup xmlns="" rendered="#{not empty field}"> 
      <nxu:set var="canSee" value="#{docSuggestionActions.getDocumentExistsAndIsVisibleWithId(field)}"> 
        <h:panelGroup rendered="#{canSee}"> 
          <nxu:set var="doc" value="#{docSuggestionActions.getDocumentWithId(field)}"> 
            <h:outputText value="#{nxd:titleOrId(doc)}"/> 
          </nxu:set> 
        </h:panelGroup>  
        <h:outputText rendered="#{not canSee}" value="#{messages['label.documentSuggestion.docNotFoundOrNotVisible']} (#{field})"/> 
      </nxu:set> 
    </h:panelGroup> 
  </div> 
</div>
